packages/mdc-button/mdc-button.scss

```diff
@@ -87,110 +57,104 @@
     border: 0;
   }
 
-  &--dense {
-    height: 32px;
-    font-size: .8125rem; // 13sp
-    line-height: 32px;
+  &:not(.mdc-ripple-upgraded) {
+    -webkit-tap-highlight-color: rgba(98, 25, 130, .3);
```

Comment:
Yep. All changes made. (I was reluctant to do so because this part will soon be rewrite in #991 )